projects/packages/forms/src/contact-form/css/file-field.css

```diff
@@ -111,6 +110,7 @@
     background-size: cover;
     border-radius: var(--jetpack--contact-form--border-radius, 0);
     outline: 2px solid  var( --jetpack--contact-form--button-primary--text-color, #FFF );
+	background-color: #333;
```

Comment:
We'll need to see if we could move away from hardcoded colours to inheriting theme.

Reply:
yeah. I agree this is not idea.

Reply:
Will look into this in a follow up PR.